hack/make-cross.sh

```diff
@@ -24,6 +24,7 @@ x ppc64le ppc64le
 x riscv64 riscv64
 GOARM=7
 export GOARM
-x arm armv7l
+# Skip building for arm due to gvisor-tap-vsock(v0.8.7) incompatibility on 32-bit ARM (see https://github.com/containers/gvisor-tap-vsock/issues/545))
```

Comment:
@AkihiroSuda can we point to main https://github.com/containers/gvisor-tap-vsock ? `32-bit ARM` issue is fixed on the main branch